auth-service/src/main/java/com/petclinic/auth/Config/SpringSecurityConfig.java

```diff
@@ -73,6 +73,7 @@ protected void configure(HttpSecurity http) throws Exception {
                 .authorizeRequests().antMatchers("/roles").hasRole("ADMIN")
                 .antMatchers(HttpMethod.POST,"/users").permitAll()
                 .antMatchers(HttpMethod.GET,"/users/*").permitAll()
+                .antMatchers(HttpMethod.GET,"/users").permitAll()
                 .antMatchers(HttpMethod.DELETE,"/users/*").permitAll()
```

Comment:
Why permit access to everyone on these endpoints?
Also, if you're going to bind POST, GET, and DELETE you might as well simplify it to
```suggestion
                .antMatchers("/users").permitAll()
```

Reply:
this has been resolve. As for the security settings, I am allowing anybody to use the API for testing for now.